Dashboards migration update (#1781)
SumoLogic/sumologic-documentation: blog-service/2023-03-02-dashboard-migration.md

---
title: Dashboard (New) Migration Tool (Beta)
image: https://www.sumologic.com/img/logo.svg
keywords:
  - sumo logic
  - service release notes
  - open source
  - dashboards new
  - dashboard migration
hide_table_of_contents: true
authors:
  - url: https://help.sumologic.com/release-notes-service/rss.xml
    image_url: /img/release-notes/rss-orange.png
---

> An automatic migration tool is now in Beta. It is available to assist you with Classic dashboard migration.  This tool can be accessed directly from a Classic dashboard page by clicking on the **Migrate to new dashboards** link toward the top of the page. 

Reviewer: ```suggestion

Beta - To prepare for Dashboards (Classic) deprecation in late 2023, we've released a tool to assist you with migrating from Dashboards (Classic) to Dashboards (New). You can access this tool directly from a Classic dashboard page by clicking on the **Migrate to new dashboards** link toward the top of the page. 
```

> <img src="/img/release_notes/service/DashboardMigration.png" alt="Migrate Dashbaord switch">

Reviewer: Fixed image link syntax and spelling in alt txt

```suggestion
<img src={useBaseUrl('img/release_notes/service/DashboardMigration.png')} alt="Migrate Dashboard switch" />
```

> For more information, including the current limitations of the tool, see [Migrate Dashboards](/docs/dashboards-new/migrate-to-new-dashboards).  We will deprecate Classic dashboard in the second half of 2023.  Start planning and migrating your Classic dashboards now. You can learn more from [our Community page](https://support.sumologic.com/hc/en-us/articles/12796506964247-All-aboard-Moving-forward-with-Sumo-Logic-s-new-dashboards).

Reviewer: ```suggestion
For more information, including the current limitations of the tool, see [Migrate Dashboards](/docs/dashboards-new/migrate-to-new-dashboards). Start planning and migrating your Classic dashboards now. You can learn more from [our Community page](https://support.sumologic.com/hc/en-us/articles/12796506964247-All-aboard-Moving-forward-with-Sumo-Logic-s-new-dashboards).
```
